packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1369,11 +1369,11 @@ export default class Meetings extends WebexPlugin {
         const meetingsCollection = this.meetingCollection.getAll();
 
         if (Object.keys(meetingsCollection).length > 0) {
-          // Some time the mercury event is missed after mercury reconnect
-          // if sync returns no locus then clear all the meetings
+          // Sometimes the mercury meetings are lost after mercury reconnect
```

Comment:
in this line we actually need to keep it as "events", there is no such thing as "mercury meetings", but we receive "mercury events" over the websocket